src/sdk/namespace_fs.js

```diff
@@ -1610,7 +1610,13 @@ class NamespaceFS {
                     await this.set_fs_xattr_op(fs_context, versioned_path, filter_fs_xattr(directory_stat.xattr), undefined, "w");
                 }
             }
-            await this._clear_user_xattr(fs_context, latest_version_dir_path, XATTR_USER_PREFIX);
+            try {
+                await this._clear_user_xattr(fs_context, latest_version_dir_path, XATTR_USER_PREFIX);
+            } catch (err) {
+                if (err.code === 'ENODATA') {
```

Comment:
Do you want to throw the error on the other case, right? (so missing an else throw or something like that...).